cmd/earth/main.go

```diff
@@ -423,6 +424,13 @@ func newEarthApp(ctx context.Context, console conslogging.ConsoleLogger) *earthA
 				},
 			},
 		},
+		{
+			Name:        "docker",
```

Comment:
I would call this something more obvious / verbose - eg `dockerfile2earthfile`. It's anyway not something used every day.